Source/WebCore/PAL/PAL.xcodeproj/project.pbxproj

```diff
@@ -522,7 +521,6 @@
 			dstSubfolderSpec = 16;
 			files = (
 				DDB04F32278E4F1B008D3678 /* libWebKitAdditions.a in Product Dependencies */,
-				DD05A36027BF0ACE0096EFAB /* libWTF.a in Product Dependencies */,
```

Comment:
PAL needs to build after WTF, because it uses WTF's headers without linking against it. So I think this dependency needs to remain in place.

Reply:
I wonder if there is a more direct way to do this. Is there a best practice other than "adding a dependency on an arbitrary file" for a header-only dependency in Xcode? I think we should keep looking for one, because I think dependencies work best when they are as close to actual "this thing reads a file" ones as possible.